gp-templates/translations.php

```diff
@@ -4,9 +4,7 @@
 	gp_project_links_from_root( $project ),
 	gp_link_get( $url, $translation_set->name ),
 ) );
-gp_enqueue_script( 'jquery-ui-core' );
-gp_enqueue_script( 'gp-editor' );
-gp_enqueue_script( 'gp-translations-page' );
+gp_enqueue_script( array( 'jquery-ui-core', 'gp-common', 'gp-editor', 'gp-translations-page' ) );
```

Comment:
Which script needs `jquery-ui-core`? Can we add it as a dependency to the script?